### PR TITLE
Fix shadow being clipped

### PR DESCRIPTION
**Pull request by revonateB0T**

### CODE REVIEW

**Comment by revonateB0T** [comment]
https://issuetracker.google.com/issues/185631200#comment3

**Comment by revonateB0T** [comment]
@FooIbar What do you think?
Should we avoid these `alpha`?

File: app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt [hunk 1]
```diff
@@ -607,19 +603,16 @@ fun GalleryListScreen(lub: ListUrlBuilder, navigator: DestinationsNavigator) = c
         val marginH = dimensionResource(id = R.dimen.gallery_list_margin_h)
         val marginV = dimensionResource(id = R.dimen.gallery_list_margin_v)
         Column(
-            modifier = Modifier.imePadding().verticalScroll(rememberScrollState())
```
Comment (FooIbar) [inline app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt:610]: I think the `Column` is not needed either?

**Review by FooIbar — APPROVED** [review]

File: app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt [hunk 2]
```diff
@@ -612,29 +612,27 @@ fun GalleryListScreen(lub: ListUrlBuilder, navigator: DestinationsNavigator) = c
                 start = contentPadding.calculateStartPadding(layoutDirection) + marginH,
                 end = contentPadding.calculateEndPadding(layoutDirection) + marginH,
                 bottom = 8.dp,
-            ),
+            ).padding(vertical = dimensionResource(id = R.dimen.search_layout_margin_v)),
```
[the remark this answers is not in the document]
Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt:615]: For better readability?

[the remark this answers is not in the document]
Comment (FooIbar) [reply app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt:615]: Ack.

**Pull request merged**